Confus/Confus/PlayerHandler.cpp

```diff
@@ -131,6 +133,14 @@ namespace Confus
 				std::make_unique<Audio::PlayerAudioEmitter>(newPlayer, &m_AudioManager)));
 		}
 		// Add self
+		int score = 0;
+		bitstreamIn.Read(score);
+		m_MainGame.getClientTeamScore().setTeamScore(ConfusShared::ETeamIdentifier::TeamRed, score);
+		bitstreamIn.Read(score);
+		m_MainGame.getClientTeamScore().setTeamScore(ConfusShared::ETeamIdentifier::TeamBlue, score);
+		int seed = 0;
+		bitstreamIn.Read(seed);
+		m_MainGame.getMazeGenerator().refillMainMazeRequest(seed, RakNet::GetTimeMS());
```

Comment:
Can't you pass a reference of the mazegenerator to the player handler instead? So that the mazegenerator holds the value of the current seed and time. Then you can also remove the inline getters in the game header file.